### ISSUE TITLE
[Bug]: `zapier build` follows invalid symlinks when copying

### ISSUE THREAD

**Issue by cscorley** [zapier/zapier-platform#706]
### Bug Description

Hi team,

I'm running into an issue with my local development environment when running `zapier build`, it will fail on the first step:

```
✖ Copying project to temp directory
 ›   Error: ENOENT: no such file or directory, stat
 ›   'hi'
```

FWIW, this file is in my .gitingore.  It would be great if the filter for this step excluded those `.gitignore` files as well.  I see it does this on zipping.  Seems like this should occur much sooner.

To be honest, I am not sure why it needs to copy into a separate directory in the first place.

Thanks!

### Reproduction Steps

1. Create a symlink to a file that does not exist: `ln -s bye hi`
2. Run `zapier build`

### Zapier Platform version

15.3.0

### Node.js version

v18.17.1

### Your Operating System

macOS 13.5.2 (22G91)

**Comment by rnegron** [comment]
Thanks for reporting this bug! I've gone ahead and created a ticket in our internal issue tracker: `PDE-4679`.

**Comment by rnegron** [comment]
Fixed by #737. We now display a warning message when detecting an invalid symlink and don't crash the build process. Thanks for reporting the issue!

**Comment by cscorley** [comment]
Thanks!
